site/content/community/guides.en.md

```diff
@@ -18,6 +18,7 @@ Share your applications, videos, and blog posts with fellow Copilots!
 | Title      | Description                          |
 | ----------- | ------------------------------------ |
 | [**Containers from the Couch series**](https://www.youtube.com/c/ContainersfromtheCouch/search?query=copilot) by <a href="https://twitter.com/realadamjkeller">@realadamjkeller</a>, <a href="https://twitter.com/brentContained">@brentContained</a>, and guests | Join Adam and Brent to learn about all the existing features of AWS Copilot with fun demos. From setting up a three-tier application with autoscaling to creating a continuous delivery pipeline with integration tests. |
+| [**AWS Copilot and another 6 ways to easily deploy apps to AWS**](https://www.youtube.com/watch?v=0lxcfJ20TwA) by <a href="https://twitter.com/Fodoj">Kirill Shirinkin</a> | Kirill Shirinkin explores the ways to easily deploy new product on AWS - including a new favourite one, AWS Copilot. |
```

Comment:
Nit: I think we should probably spell "favorite" the American way for consistency with the rest of the docs. 
```suggestion
| [**AWS Copilot and another 6 ways to easily deploy apps to AWS**](https://www.youtube.com/watch?v=0lxcfJ20TwA) by <a href="https://twitter.com/Fodoj">Kirill Shirinkin</a> | Kirill Shirinkin explores ways to easily deploy a new product on AWS - including a new favorite one, AWS Copilot. |
```